spec/acceptance/tests/yumrepo_spec.rb

```diff
@@ -58,7 +58,13 @@ def resource(host, type, name)
       resource(agent, 'file', deps_repo) do |res|
         # absent removes the entry, leaving an empty file with a known checksum
         assert_equal(res['ensure'], 'file')
-        assert_equal(res['content'], '{md5}d41d8cd98f00b204e9800998ecf8427e')
+
+        # Puppet 7 uses SHA256 as the default digest algorithm
+        if on(agent, puppet('--version')).stdout =~ %r{^7\.}
```

Comment:
you could also check `puppet config print digest_algorithm`

Reply:
yeah, thought about that, but was too late 😞